assets/src/components/account/groups/Group.tsx

```diff
@@ -1,103 +1,127 @@
 
-import { MoreMenu } from 'components/utils/MoreMenu'
-
 import { LoginContext } from 'components/contexts'
 
+import { Button, Flex, Modal } from 'honorable'
+
+import { GroupsDocument, useDeleteGroupMutation } from 'generated/graphql'
+
+import { DeleteIconButton } from 'components/utils/IconButtons'
+
 import { removeConnection, updateCache } from '../../../utils/graphql'
 
 import { Info } from '../../utils/Info'
 
 import { Permissions, hasRbac } from '../misc'
 
-import { DELETE_GROUP, GROUPS_Q } from './queries'
-
-import GroupEdit from './GroupEdit'
+import { EditGroupAttributes, EditGroupMembers } from './GroupEdit'
 import GroupView from './GroupView'
 
 export default function Group({ group, q }: any) {
   const { me } = useContext<any>(LoginContext)
   const editable = !!me.roles?.admin || hasRbac(me, Permissions.USERS)
-  const [edit, setEdit] = useState(false)
-  const [view, setView] = useState(false)
-  const [confirm, setConfirm] = useState(false)
-  const [mutation, { loading, error }] = useMutation(DELETE_GROUP, {
+  const [dialogKey, setDialogKey] = useState<
+    'confirmDelete' | 'editAttrs' | 'editMembers' | 'viewGroup' | ''
+  >('')
+
+  const [mutation, { loading, error }] = useDeleteGroupMutation({
     variables: { id: group.id },
-    onCompleted: () => setConfirm(false),
-    update: (cache, { data: { deleteGroup } }) => updateCache(cache, {
-      query: GROUPS_Q,
+    onCompleted: () => dialogKey === 'confirmDelete' && setDialogKey(''),
+    update: (cache, { data }) => updateCache(cache, {
+      query: GroupsDocument,
       variables: { q },
-      update: prev => removeConnection(prev, deleteGroup, 'groups'),
+      update: prev => removeConnection(prev, data?.deleteGroup, 'groups'),
     }),
   })
 
-  const menuItems = editable ? {
-    edit: {
-      label: 'Edit group',
-      onSelect: () => setEdit(true),
-    },
-    delete: {
-      label: 'Delete group',
-      onSelect: () => setConfirm(true),
-      destructive: true,
-    },
-  } : {
-    view: {
-      label: 'View group',
-      onSelect: () => setView(true),
-    },
-  }
-
   return (
-    <Box
-      fill="horizontal"
-      direction="row"
-      align="center"
+    <Flex
+      width="100%"
+      flexDirection="row"
+      alignItems="center"
     >
       <Info
         text={group.name}
         description={group.description || 'no description'}
       />
-      <MoreMenu onSelectionChange={selectedKey => menuItems[selectedKey]?.onSelect()}>
-        {Object.entries(menuItems).map(([key, { label, destructive }]) => (
-          <ListBoxItem
-            key={key}
-            textValue={label}
-            label={label}
-            destructive={destructive}
-            color="blue"
-          />
-        ))}
-      </MoreMenu>
+      <Flex
+        flex={false}
+        direction="row"
+        gap="large"
+        align="center"
+      >
+        {group.global && <GlobeIcon size={20} />}
```

Comment:
I think this was not accessible in console before. Only in the app.plural.sh.

Reply:
cc @dogmar

Reply:
Oh good catch. I thought I had removed all references to global, but I guess i missed one.

Reply:
It will not show anywhere, so not an issue anyways.